src/GovUk.Education.ExploreEducationStatistics.Common/Services/ImportStatusService.cs

```diff
@@ -49,11 +68,16 @@ public async Task<ImportStatus> GetImportStatus(Guid releaseId, string dataFileN
                 };
             }
 
+            var percentageComplete = CalculatePercentageComplete(import.PercentageComplete, import.Status);
```

Comment:
We be good to add tests for this service for all of its methods

Reply:
Tests added although having a problem verifying finer details of the table operations, which I've left as TODO's currently.

Reply:
Just realised I can do something like this (e.g to check the value of PercentageComplete being set) so I will go through and update the TODO's.

```
            importsTable.Verify(mock =>
                mock.ExecuteAsync(It.Is<TableOperation>(operation =>
                    operation.OperationType == TableOperationType.Replace 
                    && ((DatafileImport)operation.Entity).PercentageComplete == 100)), Times.Once);
```

Reply:
Will be nice once we get rid of all the table storage stuff and can remove stuff like this!

Reply:
Yep! 👍